.github/copilot-instructions.md

```
@@ -1,21 +1,18 @@
-Stick to core OpenGL (4.5+) with GLSL shaders, using SPIR-V.
-Minimize state changes and efficiently manage shader programs.
-Optimize the rendering loop to reduce draw calls.
-Manage resources efficiently.
-Use C++ RAII patterns (e.g., smart pointers).
-Focus on real-time performance.
-Keep GPU–CPU sync minimal and avoid unnecessary stall.
+Only add comments if the code is not self-explanatory.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)